test/standalone/test-capture-time.js

```diff
@@ -0,0 +1,109 @@
+// /**
```

Comment:
I don't think this should be considered a test at all. I think it is more of a performance benchmark that needs a bit more polish before it is instated as a requirement. I think this should be removed / moved out of the way for now.